docs/source/user_guide/input_output.md

```diff
@@ -1,31 +1,45 @@
 (target-io)=
 # Input/Output
 
-(target-formats)=
-## Supported formats
-(target-supported-formats)=
-`movement` supports the analysis of trajectories of keypoints (_pose tracks_) and of bounding box centroids (_bounding box tracks_).
+## Overview
+
+Our goal with `movement` is to enable pipelines that are input-agnostic,
+meaning they are not tied to a specific motion tracking tool or data format.
+Therefore, our input/output functions are designed to facilitate data flows
+between various third-party formats and `movement`'s own native
+[data structure](target-poses-and-bboxes-dataset) based on [xarray](xarray:).
 
-To analyse pose tracks, `movement` supports loading data from various frameworks:
-- [DeepLabCut](dlc:) (DLC)
-- [SLEAP](sleap:) (SLEAP)
-- [LightingPose](lp:) (LP)
-- [Anipose](anipose:) (Anipose)
+- [Supported third-party formats](target-supported-formats). `movement` provides convenient functions for loading/saving data in formats written by popular motion tracking tools as well as established data specifications. You can think of these as "Import" and "Export/Save As" functions.
+- [Native saving and loading with netCDF](target-netCDF). `movement` leverages xarray's built-in netCDF support to save and load datasets while preserving all variables and metadata. **This is the recommended way to save your analysis state**, allowing your `movement`-powered workflows to resume exactly where they left off.
 
-Additionally, `movement` supports loading data stored in [Neurodata Without Borders (NWB)](https://nwb-overview.readthedocs.io/en/latest/) format (using the [``ndx-pose``](https://github.com/rly/ndx-pose) extension).
+You are also welcome to try `movement` by loading some [sample data](target-sample-data) included with the package.
 
-To analyse bounding box tracks, `movement` currently supports the [VGG Image Annotator](via:) (VIA) format for [tracks annotation](via:docs/face_track_annotation.html).
+(target-supported-formats)=
+## Supported third-party formats
+
+`movement` supports the analysis of trajectories of keypoints (_pose tracks_) and of bounding box centroids (_bounding box tracks_),
+which are represented as [movement datasets](target-poses-and-bboxes-dataset)
+and can be loaded from and saved to various third-party formats.
+
+| Source Software | Abbreviation | Source Format | Dataset Type |
+|-----------------|--------------|-------------|--------------|
```

Comment:
Would it be helpful to add an extra column to state whether movement supports loading, saving, or both for each of these formats?

Reply:
Great idea! I've added such a column, with a footnote about "saving" to NumPy arrays.

![image](https://github.com/user-attachments/assets/7b171937-882f-4a03-aac9-89daf2a08b03)

Reply:
Excellent